DEVGUIDE.md

```diff
@@ -147,6 +147,12 @@ If you are on a Mac, you can run this command from the root of the repository:
 sh build.sh -c Release
 ```
 
+Or if you are on Linux:
+
+```shell
+./build.sh -c Release
+```
+
```

Comment:
Probably just merge "on Linux or Mac"?

Reply:
The Mac instruction (`sh build.sh -c Release`) did not work on Ubuntu for me

Reply:
Huh, ok, let's keep a separate one in this case.